packages/ra-ui-materialui/src/detail/CreateView.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import { Children, cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import { CreateControllerProps } from 'ra-core';
+import { Card } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import classnames from 'classnames';
+import { CreateProps } from '../types';
+import { TitleForRecord } from '../layout';
+
+export const CreateView = (props: CreateViewProps) => {
+    const {
+        actions,
+        aside,
+        basePath,
+        children,
+        classes: classesOverride,
+        className,
+        component: Content,
+        defaultTitle,
+        hasList,
+        hasShow,
+        record = {},
+        redirect,
+        resource,
+        save,
+        setOnSuccess,
+        setOnFailure,
+        setTransform,
+        saving,
+        title,
+        version,
+        ...rest
+    } = props;
```

Comment:
CreateView should use the CreateContext to get the controller props. See how I did in ListView